Documentation/gettingstarted/namespace-cilium.rst

```diff
@@ -0,0 +1,9 @@
+Specify Environment Variables for Hubble
```

Comment:
```suggestion
Specify Environment Variables
```
I think that the "for Hubble" part can be omited as there is nothing Hubble specific in these instructions.

Reply:
done